Web/Edubase.Web.UI/Views/Shared/EditorTemplates/GovernorViewModel.cshtml

```diff
@@ -81,19 +81,19 @@ else
         <div class="govuk-form-group @Html.ValidationCssClassFor(x => x.FirstName)">
             @Html.LabelFor(x => x.FirstName, $"First name{additionalInfo}", new { @class = "govuk-label" })
             @Html.ValidationMessageFor(x => x.FirstName, null, new { @class = "govuk-error-message" })
-            @Html.TextBoxFor(x => x.FirstName, new { @class = string.Concat("govuk-input ", Html.TextBoxValidationClass(x => x.FirstName)) })
+            @Html.TextBoxFor(x => x.FirstName, new { @class = string.Concat("govuk-input ", Html.TextBoxValidationClass(x => x.FirstName)), id = "governorFirstNameInput" })
```

Comment:
We've been using the naming convention of dashes between words and lower casing so could you change all the ids here to that i.e. "governor-first-name-input".
Also, on the note of the use of 'input' I have been using 'textbox' when the element is as such - thoughts?

Reply:
That should be in line with our naming convention now, let me know what you think @antk43